pyflix2/pyflix2.py

```diff
@@ -427,7 +427,7 @@ def get_movie_by_title(self, movie_title):
                 return movie
         return None
 
-    def get_catalog(self, catalog_type='index', chunk_size=4096):
+    def get_catalog(self, catalog_type='index', raw=False, chunk_size=4096):
```

Comment:
Do you mind keeping the methods argument backward-compitable by adding the new argument at the end? Same comment for the other get_catalog method.